modules/__init__.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+from . import bomb_word
+from . import chat
+from . import infometr
+from . import sasai
+from . import say
+from . import timer
+
+
+def get_handlers():
+    from functools import reduce
+    modules = [bomb_word, chat, infometr, sasai, say, timer]
+    handlers = [getattr(m, 'handlers', []) for m in modules]
+    return reduce(lambda x, y: x + y, handlers)
```

Comment:
if i understand correctly, that's list flattening? maybe better do it with list comprehension, i.e.:
`
return [handler for m in modules for hander in getattr(m, 'handlers', [])]
`

Reply:
good point, will fix